.github/renovate.json

```
@@ -13,5 +13,5 @@
             "separateMajorMinor": "false"
         }
     ],
-    "schedule": ["on the third day of the month"]
+    "schedule": ["the 3rd day of the month"]
```

```suggestion
    "schedule": ["on the 3rd day of the month"]
```
https://codepen.io/antonlydike/full/Kgygpr indicates that this is necessary.